src/migrations/20240126095544-add-flag-trends.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+exports.up = function(db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS flag_trends (
+            id VARCHAR(255) NOT NULL,
+            project VARCHAR(255) NOT NULL REFERENCES projects (id) ON DELETE CASCADE,
```

Comment:
I'm thinking whether we should delete trends data when we delete a project. My assumption is that when I delete a project I want to remove its traces in the trends. In the original PR it was no action (https://github.com/Unleash/unleash/pull/5914/files)